cime_config/tests.py

```diff
@@ -166,11 +166,11 @@
             "SMS_Ld2.ne30_oECv3.BGCEXP_CNTL_CNPRDCTC_1850.elm-bgcexp",
             "SMS_D_Ld1.T62_oEC60to30v3.DTESTM",
             "SMS_D_Ld1.ne30pg2_r05_EC30to60E2r2.A_WCYCL1850S_CMIP6",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_VT",
+            "ERP_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_fixed_subcycle",
+            "ERS_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_VT",
             "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_ESMT",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFOMP",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMF1-RCEMIP",
+            "ERP_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFOMP.eam-mmf_fixed_subcycle",
```

Comment:
This has build errors on Theta (and Chrysalis) due to threading flags (ERP forces a threaded build). Probably best to keep it as `ERS` until OpenMP offload is fixed for CPU-only runs with Intel.

Reply:
Ah, I didn't realize ERP would force threading. Why is that the case? 
Either way, I'll revert this test to ERS in a new PR.